### PR TITLE
Updating swagger spec to latest

### PR DESCRIPTION
**Pull request by nikhiljindal**
Fixes https://github.com/kubernetes/kubernetes/issues/14048

PR generated by running:

```
./swagger-spec/update.sh
```

cc @bgrant0607 @brendandburns

### CODE REVIEW

File: swagger-spec/api/index.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "swaggerVersion": "1.2",
   "apiVersion": "",
-  "basePath": "https://127.0.0.1:6443",
+  "basePath": "https://10.10.10.10:6443",
```
Comment (bgrant0607) [inline swagger-spec/api/index.html]: Is this field used by anyone?

**Comment by bgrant0607** [comment]
Why isn't this automatically updated? Do we also need to generate docs?

File: swagger-spec/api/index.html (hunk 1 above)
Comment (nikhiljindal) [inline swagger-spec/api/index.html]: Swagger-ui was using it, but we made that read-only so it is not using this any longer.
I dont know of any other usage of this field.

**Comment by nikhiljindal** [comment]
https://github.com/kubernetes/kubernetes/issues/4763 tracks updating spec on gh-pages automatically.

Good point about updating docs.
We have docs at two places:
* At Head: Filed https://github.com/kubernetes/kubernetes/issues/14069 to ensure that HEAD api-reference docs are updated each time swagger-spec is updated.
* At kubernetes.io: We surface docs from 1.0 on gh-pages. To update that, we will need to cherrypick the PR that changes the spec in 1.0 branch and then regenerate the doc.
Will request cherrypicking https://github.com/kubernetes/kubernetes/pull/10426

File: swagger-spec/api/index.html (hunk 1 above)
Comment (lavalamp) [inline swagger-spec/api/index.html]: Why the switch to 10.10.10.10?

Comment (nikhiljindal) [inline swagger-spec/api/index.html]: This PR just syncs the swagger-spec on gh-pages with the one on HEAD.
Here is the PR that changed this on HEAD: https://github.com/kubernetes/kubernetes/pull/10713

Comment (lavalamp) [inline swagger-spec/api/index.html]: Oh, gotcha. Didn't notice this was gh-pages.

**Comment by lavalamp** [comment]
LGTM

**Comment by lavalamp** [comment]
Auto-updating this would be a good thing to have a script do ~daily.

**Comment by lavalamp** [comment]
Heh... the merge queue can't handle gh-pages. Manually merging.

**Comment by lavalamp** [comment]
Hm actually before I merge-- @nikhiljindal does this include any non-v1.0.x changes? You produced this from the 1.0 release branch, right?

**Comment by nikhiljindal** [comment]
This is generated from HEAD.
We expose HEAD swagger-spec at kubernetes.io/third_party/swagger-ui/

**Comment by lavalamp** [comment]
@nikhiljindal But shouldn't the stuff in gh-pages be 1.0?

**Comment by lavalamp** [comment]
IRL chat: we keep swagger from HEAD in gh-pages; we need to start also storing the swagger from e.g. 1.0.6. @nikhiljindal will file an issue about this.

**Pull request merged**